tests/FsCodec.NewtonsoftJson.Tests/PicklerTests.fs

```diff
@@ -56,12 +56,12 @@ module CartV2 =
         | Create of CreateCart
         interface IUnionContract
 
-let [<Fact>] ``Deserilize expected null into optional property`` () =
+let [<Fact>] ``Deserialize missing field a as optional property None value`` () =
     let expectedCreateCartV2: CartV2.CreateCart =  { Name = "cartName"; CartId = None }
     let createCartV1: CartV1.CreateCart =  { Name = "cartName" }
 
     let createCartV1JSON = JsonConvert.SerializeObject createCartV1
 
     let createCartV2 = JsonConvert.DeserializeObject<CartV2.CreateCart>(createCartV1JSON)
 
-    test <@ createCartV2 = expectedCreateCartV2 @>
\ No newline at end of file
+    test <@ createCartV2 = expectedCreateCartV2 @>
```

Comment:
even in unquote, convention is to stick to exected = actual order like xunit asserts